src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -150,10 +150,13 @@ public string Filename
 
         /// <summary>
         /// Gets or sets the full path of the file containing the matching line.
+        /// </summary>
+        /// <remarks>
+        /// It will be "InputStream" if the object came from the input stream.
+        /// </remarks>
```

Comment:
This looks like a duplicate of the remark below.

Reply:
Fixed